packages/web/src/utils/menus/utils.ts

```diff
@@ -0,0 +1,198 @@
+/**
+* Handle event for keyboard control of menu
+* @param {Event} event
+* @param {Element} menu
+* @param {any[]} queue
+*/
+export async function handleKeyDownMenu(event, menu, queue) {
+  let key = event.keyCode || event.which;
```

Comment:
Can you replace this with: `let key = event.key;`? I think `event.key` and  `event.which` have been deprecated.